wondrous-app/graphql/fragments/org.ts

```diff
@@ -26,6 +26,7 @@ export const OrgFragment = gql`
     podCount
     category
     shared
+    layout
```

Comment:
there's a performance hit fom this, can we just add it specifically to 
`export const GET_ORG_FROM_USERNAME = gql`
  query getOrgFromUsername($username: String!) {
    getOrgFromUsername(username: $username) {
      ...OrgFragment
    }
  }
  ${OrgFragment}
`;
`